public/javascripts/scripts.js

```diff
@@ -0,0 +1,26 @@
+
+let options = {
+    strings: ['"kipeet"'],
+    typeSpeed: 60,
+    // time before typing starts
+    startDelay: 2500,
+    // backspacing speed
+    // backSpeed: 30,
+    // time before backspacing
+    // backDelay: 5000,
+    // loop
+    // loop: false,
+    // false = infinite
+    // loopCount: 5,
+    // show cursor
+    showCursor: false,
+    // character for cursor
+    cursorChar: "|",
+    // attribute to type (null == text)
```

Comment:
A comment line should be easier to understand. thats why its a comment. Lay man, might not know==

Reply:
Ok sir